react/features/video-status-label/components/VideoStatusLabel.js

```diff
@@ -1,9 +1,21 @@
+import { Tooltip } from '@atlaskit/tooltip';
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
-import { toggleAudioOnly } from '../../base/conference';
 import { translate } from '../../base/i18n';
 
+declare var interfaceConfig: Object;
+
+/**
+ * Whether or not the video quality button is displayed in a menu.
+ *
+ * @private
+ * @type {boolean}
+ */
+const HAS_VIDEO_QUALITY_BUTTON
+    = (interfaceConfig.TOOLBAR_BUTTONS || [])
+        .find(button => button === 'videoquality');
```

Comment:
It looks weird to me that the VideoStatusLabel somehow "knows" the name of the "videoquality" button. We should either merge those 2 features into 1 if they're so much related or make this check in a different more independent way.